apps/frontend/src/context/DataContext.tsx

```diff
@@ -20,8 +20,13 @@ const DataContext = React.createContext<{
   stop: () => void;
   addLap: () => void;
   isRunning: boolean;
+  activeRoute: Waypoint[];
```

Comment:
could be some small confusion around the names.
activeRoute seems to me like a Route
and activeRouteName like a String

i guess activeRouteWaypoints could be an option

Could also maybe package these 2 fields into a type, because they seem to belong togheter.

I would also maybe try to move a lot of the Route Point svg details into a separate file

Reply:
Yep, made `activeRoute` an object with `name` and `waypoints` and moved stuff to `gps.ts`!